dvc/command/gc.py

```diff
@@ -39,9 +40,9 @@ def run(self):
             return 1
 
         self.repo.gc(
-            all_branches=self.args.all_branches,
-            all_tags=self.args.all_tags,
-            all_commits=self.args.all_commits,
+            remove_all_tags=self.args.remove_all_tags,
```

Comment:
Opted for explicit flags to avoid confusion such as previous `--remove=tags,commits`(where it is not clear what `commits` are and why you are not able to pass specific tag or branch to it) and also to simplify completion.